uint16 state #小车状态机
uint16 path_num #第几条线路
bool is_arrive_goal #是否到达终点
float64 speed #小车速度
